internal/pkg/template/workload.go

```diff
@@ -203,6 +203,17 @@ type LogConfigOpts struct {
 	Secrets        map[string]Secret
 }
 
+// HTTPTargetContainer represents the target group of a load balancer that points to a container.
+type HTTPTargetContainer struct {
+	Name string // Name of the container.
```

Comment:
qq: where is `Name` used?

Reply:
nowhere atm, I just did it for consistency but I can remove it since it's not used 😅